modules/monitoring-setting-log-levels-for-monitoring-components.adoc

```diff
@@ -19,7 +19,7 @@ The default log level is `info`.
 
 .Prerequisites
 
-* *If you are setting a log level for Prometheus Operator, Prometheus, or Thanos Querier in the `openshift-monitoring` project*:
+* *If you are setting a log level for Alertmanager, Prometheus Operator, Prometheus,or Thanos Querier in the `openshift-monitoring` project*:
```

Comment:
`Prometheus,or Thanos Querier`
change to
`Prometheus, or Thanos Querier`

Reply:
fixed